assets/dev/js/editor/regions/responsive-bar/view.js

```diff
@@ -0,0 +1,100 @@
+export default class View extends Marionette.ItemView {
+	getTemplate() {
+		return '#tmpl-elementor-templates-responsive-top-bar';
+	}
+
+	className() {
+		return 'e-mq-bar';
+	}
+
+	ui() {
+		const prefix = '.e-mq-';
+
+		return {
+			switcherOption: prefix + 'switcher__option',
```

Comment:
Why not hardcoded?

Reply:
So It can easily be refactored when @matipojo reviews it.

Reply:
Anyway, you need to perform a search-replace on the whole code.